src/test/analyzer_test.ts

```diff
@@ -84,6 +85,32 @@ suite('Analyzer', () => {
           ['MyNamespace.SubBehavior', 'MyNamespace.SimpleBehavior']);
     });
 
+    // TODO(fks) 09-28-2016: Fix off-by-one error in warning sourceRange line
+    // numbers.
+    test('throws when cant find behavior', async() => {
```

Comment:
I find targeted tests easier to debug when they break, especially if you can get an idea from which tests broke what went wrong. I'd love to test Polymer-specific things in the Polymer-specific tests. We should create some analyzer setup utilities for tests on various plugins. This is fine for now though.